src/streaming/ManifestUpdater.js

```diff
@@ -249,6 +249,7 @@ function ManifestUpdater() {
             eventBus.trigger(Events.DYNAMIC_TO_STATIC);
             isUpdating = false;
             isStopped = true;
+            eventBus.trigger(Events.MANIFEST_UPDATED, { manifest: manifest });
```

Comment:
@bbert As you implemented the dynamic to static logic originally: Was there a reason to not trigger this event for the final update?

Reply:
I guess it was to not update the manifest and then re-compose the streams and the segment timelines, in order to maintain the latest live manifest, since as it comes to static the timestamps starts from 0 (based on `presentationTimeOffset`).
But player should not request anymore segments once end of dynamic timeline is reached.